docs/trino-connector/supported-catalog.md

```diff
@@ -14,7 +14,6 @@ The catalogs currently supported by the Apache Gravitino connector are as follow
 
 ## Create catalog
 
-Trino itself does not support creating catalogs. 
```

Comment:
@diqiu50
Please help to clarify it, thanks.

Reply:
In the current minimum support version and required, removing this sentence is ok.